CHANGELOG/CHANGELOG-0.6.md

```diff
@@ -0,0 +1,18 @@
+# Changelog 0.6
+
+## 0.5.0 - 2023-05-24
+
+### Added
+
+- Added Theming and Coloring in Service Dependency Graph. ([#187](https://github.com/antrea-io/theia/pull/187), [@Dhruv-J])
+- Group Pods by Pod Labels in Service Dependency Graph. ([#200](https://github.com/antrea-io/theia/pull/200), [@Dhruv-J])
+- Added coverage and threshold for unit tests and python tests. ([#171](https://github.com/antrea-io/theia/pull/171), [@Dhruv-J]) ([#296](https://github.com/antrea-io/theia/pull/296), [@Dhruv-J])
```

Comment:
Not sure if it's important to list here, it's not something users will leverage!